server/src/server.ts

```diff
@@ -0,0 +1,26 @@
+import express, { NextFunction, Request, Response } from 'express'
+import { routes } from './routes'
+
+const PORT = process.env.PORT
+
+const app = express()
+
+app.use(express.json())
+
+app.use(routes)
+
+app.use(
+  (err: Error, request: Request, respose: Response, next: NextFunction) => {
```

Comment:
I recommend extracting this function into a file or a separate function to improve readability

Reply:
Ajusted